kratos/input_output/stl_io.cpp

```diff
@@ -217,7 +218,20 @@ void StlIO::ReadSolid(
     KRATOS_ERROR_IF(word != "solid") << "Invalid stl file. Solid block should begin with \"solid\" keyword but \"" << word << "\" was found" << std::endl;
     std::getline(*mpInputStream, word); // Reading solid name to be the model part name
 
-    word.erase(word.begin(), std::find_if(word.begin(), word.end(), [](int ch) {return !std::isspace(ch);})); // Triming the leading spaces
+    // Remove Comments
+for (const auto& symbol : {"COMMENT:", ";"}) {
```

Comment:
Is it not possible to just remove any word after the first one? I have the impression this is very specific for one case